.github/workflows/trailing_commas.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+"""This is a prettify script so that flutter formatting can work efficiently.
+
+Methodology:
+    Analyses the `lib` and `test` directories to go through the code line by
+    line and add commas if the syntax matches the conditions.
+    This script was created to help improve code quality by making it prettier
+    and more readable.
+
+NOTE:
+    This script complies with our python3 coding and documentation standards
+    and should be used as a reference guide. It complies with:
+        1) Pylint
+        2) Pydocstyle
+        3) Pycodestyle        
+    Run these commands from the CLI to ensure the code is compliant for all
+    your pull requests if you want to contribute to this file.
+
+"""
+
+# Standard imports
+import os
+import sys
+import argparse
+
+
+def arg_parser_resolver():
+    """Resolve the CLI arguments provided by the user.
+
+    Args: None
+    Returns: result: Parsed argument object
+
+    """
+    parser = argparse.ArgumentParser(
+        description='for parsing across the directory')
+    parser.add_argument('--dir', type=str, required=False, default=os.getcwd(),
+                        help='directory-location where files are present')
+    return parser.parse_args()
+
+
+def syntax_matcher(root: str, files: list):
+    """Pattern matching and adding trailing commas at the required places.
+
+    Args: 
+     - root -> Name of the root directory
```

Comment:
Make the definition of the arguments in the doctring exactly like that specified in the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#244-decision) in all cases.